<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>811</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item row="1" column="1">
      <widget class="QCalendarWidget" name="calendarWidget_from"/>
     </item>
     <item row="2" column="1">
      <widget class="QCalendarWidget" name="calendarWidget_to"/>
     </item>
     <item row="0" column="0" rowspan="3">
      <widget class="QTableView" name="tableView"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="bookingNummer">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFunktionen">
    <property name="title">
     <string>Funktionen</string>
    </property>
    <addaction name="actionDatei_einlesen"/>
    <addaction name="actionBuchung_anlegen"/>
    <addaction name="actionRefresh"/>
   </widget>
   <addaction name="menuFunktionen"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDatei_einlesen">
   <property name="text">
    <string>Datei einlesen</string>
   </property>
  </action>
  <action name="actionBuchung_anlegen">
   <property name="text">
    <string>Buchung anlegen</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
